index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PraisyDram 2022</title>
+  <link
+    rel="stylesheet"
+    href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+  />
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <main>
+    <header class="header">
+      <i class="menu-icon fas fa-bars fa-2x"></i>
+      <a href="./index.html"><img src="./public/assets/icons/pd-logo.png" alt="PraisyDram Logo"></a>
+      <nav class="menu-items">
+        <i class="close fas fa-times"></i>
+        <ul>
+          <li><a href="./about.html" >About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li class="last-list-child"><a href="#">Concert Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="headline">
+      <div class="headline-title">
+        <p>"ITI-CCR KUNST Presents!"</p>
+        <h1>PRAISYDRAM 2022</h1>
+      </div>
+      <p class="headline-details">A wonderful opportunity to witness the greatness of God in our lives as his most inportant creation.
+        Fun packed activities to cherish for the rest of your live. You just can't miss this!
+      </p>
+      <div class="headline-bottom">
+        <p class="bottom-top">
+          2022.03.18(FRI) ~ 19(SAT)
+        </p>
+        <p>
+          @ KNUST Parade Grounds
+        </p>    
+      </div>    
+    </section>
+    <section class="main-program">
+      <h2 class="program-title">Main Programs</h2>
+      <span class="border-span"></span>
+      <div class="program-cards">
+        <article class="card">
+          <i class="fas fa-microphone-alt fa-2x"></i>
```

Comment:
- [ ] Your card's design is different from the template, Please make sure the color you are using is a little transparent and it matches the design on the template.
- [ ] When we hover the mouse over cards a white border around cards should appear as on the design, Please apply hover effects to your cards so the cards show a white border when we hove mouse over them.

### Your design

![image](https://user-images.githubusercontent.com/83803414/157689914-19199c74-d6e1-42d3-8e49-546a467903eb.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/157689981-91c089cf-d233-49a4-b0d5-3e74f7a6d17b.png)